NGINXaaS: Address missing info and points of user friction (#1588)
nginx/documentation: content/nginxaas-google/monitoring/access-management.md

> F5 NGINXaaS for Google Cloud (NGINXaaS) leverages Workload Identity Federation (WIF) to integrate with Google Cloud services. For example, when WIF is configured, NGINXaaS can export logs and metrics from your deployment to Cloud Monitoring in your chosen Google project. To learn more about WIF on Google Cloud, see [Google's Workload Identity Federation documentation](https://cloud.google.com/iam/docs/workload-identity-federation).

Reviewer: I know this was just a white-space change, but since you're already here, may I suggest the following simplification:
```suggestion
F5 NGINXaaS for Google Cloud (NGINXaaS) uses Workload Identity Federation (WIF) to integrate with Google Cloud services. For example, with WIF configured, your NGINXaaS deployment can export logs and metrics to Cloud Monitoring in your Google project. To learn more, see [Google's Workload Identity Federation documentation](https://cloud.google.com/iam/docs/workload-identity-federation).
```

Author: fixed it.

## What's next

> [Monitor your deployment]({{< ref "/nginxaas-google/monitoring/enable-monitoring.md" >}})

Author: fixed it.
